frontend/simple/assets/sass/bulma_overrides/elements/_index.scss

```diff
@@ -1,5 +1,5 @@
 @import "../../node_modules/bulma/sass/elements/box";
-@import "../../node_modules/bulma/sass/elements/button";
+@import 'button';
```

Comment:
is there a reason for mixing single and double quotes in this file?

Reply:
Not at all, it's just the linter doesn't have that rule. Added. `'string-quote: double'`